app/models/topic_user.rb

```diff
@@ -312,26 +312,22 @@ def update_last_read(user, topic_id, post_number, new_posts_read, msecs, opts =
         threshold: SiteSetting.default_other_auto_track_topics_after_msecs
       }
 
-      # 86400000 = 1 day
```

Comment:
No idea what this comment is referring to.

Reply:
I think it is because there is an `msecs` argument for this function and there was a bigger comment closer to the top:

https://github.com/discourse/discourse/commit/f0e942f6474ffa41be835e2deda9ebdac2f6c93c#diff-94a17ebd5fec7b95f6eb1bb1576a8e15c52f992367c56e94640a8834b31615a9R168-R188

But yeah not really necessary